types/eslint/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private": true,
+    "exports": {
+        ".": {
+            "types": "./index.d.ts"
+        },
+        "./use-at-your-own-risk": {
+            "types": "./unsupported.d.ts"
+        },
+        "./rules": {
+            "types": "./rules/index.d.ts"
+        }
```

Comment:
this export does not exist

https://github.com/eslint/eslint/blob/6b85426c33ba7ac0206cccef39ccc875b773aeae/package.json#L10-L14

Reply:
Yep - `eslint` doesn't expose anything like this, but we just export a single type at this location, and have forever. Since type imports are elided, it should be fine - I saw no reason to break this entrypoint.